listing-rewards/program/tests/listing_rewards_test/mod.rs

```diff
@@ -4,6 +4,7 @@ use solana_program_test::*;
 
 pub const TEN_SOL: u64 = 10_000_000_000;
 pub const ONE_SOL: u64 = 1_000_000_000;
+pub const TEST_COLLECTION: &str = "Cehzo7ugAvuYcTst9HF24ackLxnrpDkzHFajj17FuyUR";
```

Comment:
Oh test collection is already in the testing module. Drop it from program constants.